Project-Code/src/index.js

```diff
@@ -59,26 +58,36 @@ const user = {
 
 
 app.get('/', (req, res) =>{
-    
     res.render('pages/login')
 });
 
 app.get('/test', (req, res)=>{
+    //Making db query to retrieve address before API call to get voterInfo
+    const query = "SELECT addressLine1, addressLine2, city, state, zip_code FROM voters WHERE username='test';";
+    const values = [user];
+
+    db.any(query, values)
+        .then(async (data) =>{
+            address = data.addressLine1 + data.city + data.state + data.zip_code;
+      console.log(data);
+      console.log(address);
+      console.log(data.addressLine1);
+        })
+        .catch((error) => {
+                //hanlde errors
+                console.log(error);
+                res.redirect("/");
+        });
+    //API call
+    console.log(values);
     axios({
-         url: `https://www.googleapis.com/civicinfo/v2/voterinfo?address=2845%20trinity%20loop%20broomfield%20CO&includeOffices=true&levels=regional&roles=governmentOfficer&key=${process.env.API_KEY}`,
+            url: `https://civicinfo.googleapis.com/civicinfo/v2/voterinfo?address=${user.address}&key=${process.env.API_KEY}`,
```

Comment:
Read from `process.env` at the top of the file. Reading from `process` is slow. Add it like this near the top

```js
const { API_KEY } = process.env
```